main/reference/zoe-api/user-seat.md

```diff
@@ -62,21 +71,35 @@ A **Payout** is a **Payment** that goes to a party in a successful transaction,
 escrowed assets in accordance with the result of the transaction. Returns a **Promise** for the **Payout**
 **Payment** associated with the *keyword* argument.
 
+This promise will be resolved when the seat exits.
+
 ## E(UserSeat).getOfferResult()
   - Returns: **Promise&lt;OfferResult>**
 
 Returns a **Promise** for an **OfferResult**. The **OfferResult** can be literally anything. 
-For example, in tests
-for the Automated Refund Dapp, it's the string "The offer was accepted". In
+For example, in tests for the Automated Refund Dapp, it's the string "The offer was accepted". In
 the Covered Call example, it's a call option, which is an assayable **[Invitation](./zoe-data-types.md#invitation)**
 to buy the underlying asset. Strings and invitations are the most common things returned.
 The value is set by the returned result of the **offerHandlers** function passed
 as an argument to **[zcf.makeInvitation()](./zoe-contract-facet.md#zcf-makeinvitation-offerhandler-description-customproperties-proposalshape)**.
 
+Since the contract can return whatever it wants as an offerResult, there is no guarantee that the
+promise will resolve promptly.
+
 ## E(UserSeat).hasExited()
   - Returns: **Promise&lt;Boolean>**
 
-Returns **true** if the seat has exited, **false** if it's still active.
+Returns **true** if the seat has exited, **false** if it's still active. The value is returned
+immediately.
+
+If you want to take some action when the seat does exit, use **getExitSubscriber()**
+and use
+```
+const subscriber = E(seat).getExitSubscriber();
+E.when(E(subscriber).getUpdateSince(), () => takeAction());
+```
```

Comment:
The difference being that subscribing to a remote contract remains valid across upgrade of the contract, whereas a remote promise to be resolved by a contract will instead be broken by an intervening contract upgrade.